app/javascript/components/patient/Patient.js

```diff
@@ -238,6 +239,19 @@ class Patient extends React.Component {
               {this.renderEditLink('Contact Information', 2)}
             </div>
             <div className="item-group">
+              {this.props.details.date_of_birth && isMinor(this.props.details.date_of_birth) && (
+                <React.Fragment>
+                  <span className="text-danger">Monitoree is a minor{!this.props.details.head_of_household && this.props.hoh && '. '}</span>
+                  {!this.props.details.head_of_household && this.props.hoh && (
+                    <div>
+                      View contact info for head of household:
+                      <a className="pl-1" href={patientHref(this.props.hoh.id, this.props.workflow)}>
```

Comment:
can we add a `target="_blank"` to this href? that way if the user clicks it, a new tab opens and they have both sets of info accessable

Reply:
@sgober As far as I can tell the only places we do that are when linking to external sites. I'm thinking it might be best to just leave this as is for consistency/user expectations. Savvy users can always cmd+click or right click and open in new tab if they want.

Reply:
we do this in the apply to household modal so the user isn't redirected away from the modal.  I think in this situation that approach might also be valid but don't have a strong opinion.  Might be worth asking the PO team?

Reply:
Ah missed that one. I think doing `target="_blank"` makes more sense in a modal, to protect the user from effectively losing whatever progress they've made during their interaction with that modal. In this case, there is no possibility that the user is in a similar "work in progress" state, and so I would prefer to leave it as is.

@sgober If you think this deserves more consideration, you can of course still feel free to post a question for the PO team (in which case just link to it from here). Otherwise, let's leave it as is.

Reply:
we can leave it as is then